src/utils/AxiosInstance.ts

```diff
@@ -7,15 +7,9 @@ import { SessionStorageKeys } from 'sbc-common-components/src/util/constants'
  */
 
 // this is a function because a getter doesn't work reliably on app start
-function getAccountId (): string {
-  // if we can't get account id from ACCOUNT_ID
-  // then try to get it from CURRENT_ACCOUNT
-  let accountId = sessionStorage.getItem('ACCOUNT_ID')
```

Comment:
In my testing, ACCOUNT_ID was always empty, so I stopped checked for it.

CURRENT_ACCOUNT seems to be consistently populated.

This also removed one place where `ACCOUNT_ID` might not match `CURRENT_ACCOUNT.id`.